Doc/using/mac.rst

```diff
@@ -17,7 +17,7 @@ the IDE and the Package Manager that are worth pointing out.
 Getting and Installing MacPython
 ================================
 
-Mac OS X 10.8 comes with Python 2.7 pre-installed by Apple.  If you wish, you
+macOS 10.8 comes with Python 2.7 pre-installed by Apple.  If you wish, you
```

Comment:
It was Mac OS X.

Reply:
Current versions of macOS still have Python 2.7 I believe